ApollosProject/apollos-prototype
# Hooks up text feature

![2019-07-08 09 14 33](https://user-images.githubusercontent.com/1637694/60814892-7c8e1a80-a164-11e9-80a8-bf621a2902f5.gif)

## DESCRIPTION

### What does this PR do, or why is it needed?
Hooks up the ActionCard component as the `TextFeature` in the `ContentSeriesContentItem`. Along the way, does the following things. 

- Exports `TextFeature` from `ui-kit`
- Adds a `LikeButton` component that fetches it's own data, and uses that new component inside the ActionContainer. 
- ~Adds a new view for the `ContentSeriesContentItem`, which right now is just the `UniversalContentItem` + a `Features` component at the bottom.~
  - Added `Features` to to the universal content item (shows nothing if there are no features or the content item does not support features) 

### What design trade-offs/decisions were made?

Added the `LikeButton`, to simplify data fetching for the "Like" data. 

### How do I test this PR?

1. Open the Content Item called the "New Weekend Experience"
2. You should see a TextFeature after the main body content. 

### What automated tests did you write?
Will write some snaps. 

## TODO:

- [x] Closes #838 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
